ember-file-upload/src/test-support.ts

```diff
@@ -91,6 +91,60 @@ export async function dragAndDrop(
   return triggerEvent(dropzone, 'drop', { dataTransfer });
 }
 
+interface FileSystemEntryStub {
+  isFile: boolean;
+  file: (callback: (file: File | Blob) => void) => void;
+}
+
+export async function dragAndDropDirectory(
+  selector: string,
+  folderName: string,
+  filesInDirectory: (File | Blob)[],
+  ...singleFiles: (File | Blob)[]
+) {
```

Comment:
I wonder if we should use an options object here. From the call site the two types of files (in directory and single) are not very clear.

Reply:
Fair call, it's not super clear indeed. Would you make 1 `opts` object? Or keep the `selector` and/or `folderName` and make an object for the rest?

I'm doubting if the `folderName` even needs to be here, but that'll probably be relevant for the bigger remark about the directory itself being reported as `UploadFile`